docs/extensions.md

```diff
@@ -1,4 +1,4 @@
-## Extending `build-harness` with targets from another repo
+# Extending `build-harness` with targets from another repo
```

Comment:
Aha, we should presumably exclude this file then as well from listing. It will be picked up by the README.md lint.

Reply:
I added such ignore file in this commit:
https://github.com/cloudposse/build-harness/pull/230/commits/31362107d0fdd97964c1f31071f98a76915a863b